r/tools/linuxlibs.R

```diff
@@ -21,28 +21,19 @@ dst_dir <- paste0("libarrow/arrow-", VERSION)
 
 arrow_repo <- "https://dl.bintray.com/ursalabs/arrow-r/libarrow/"
 apache_src_url <- paste0(
-  "https://www.apache.org/dyn/closer.cgi/arrow/arrow-", VERSION,
+  "https://archive.apache.org/dist/arrow/arrow-", VERSION,
```

Comment:
ASF release policies advise against this -- maybe let's open a JIRA about making the mirror resolution more robust?